src/components/MultiStoryContainer/hooks/useDraggableGesture.ts

```diff
@@ -76,13 +83,25 @@ const useDraggableGesture = ({
         scale.value = withTiming(
           0,
           {
-            duration: 300,
+            duration: speed,
           },
           () => {
             isCompleted.value = true;
             isLongPressed.value = false;
           }
         );
+        translateX.value = withTiming(
```

Comment:
@prince-d-simform It should be as follows
Center Story on X-axis and Center Story on Y-axis